pkg/policies/security/security.go

```diff
@@ -114,29 +112,27 @@ func check(ctx context.Context, rep repositories, c *github.Client, owner,
 		Bool("enabled", enabled).
 		Msg("Check repo enabled")
 
-	fc, _, rsp, err := rep.GetContents(ctx, owner, repo, filePath, nil)
-	if err != nil {
-		if rsp != nil && rsp.StatusCode == http.StatusNotFound {
-			return &policydef.Result{
-				Enabled:    enabled,
-				Pass:       false,
-				NotifyText: "SECURITY.md not found.\n" + fmt.Sprintf(notifyText, owner, repo),
-				Details: details{
-					Exists: false,
-					Empty:  true,
-				},
-			}, nil
-		}
+	var q struct {
+		Repository struct {
+			SecurityPolicyUrl       string
```

Comment:
Should these be `githubv4.String` and `githubv4.Boolean`?

Reply:
https://github.com/shurcooL/githubv4#scalar-types
> However, depending on how you're planning to use the results of your query, it's often more convenient to use other Go types.
> The encoding/json rules are used for converting individual JSON-encoded fields from a GraphQL response into Go values. See https://godoc.org/encoding/json#Unmarshal for details. The json.Unmarshaler interface is respected.
> That means you can simplify the earlier query by using predeclared Go types:

Using `string` is more convenient, here, don't need to check the pointer for nil, etc.

Reply:
SGTM.